Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -82,6 +79,8 @@ public class NodeSettings
         /// <summary>URI to node's API interface.</summary>
         public Uri ApiUri { get; set; }
 
+        public TextFileConfiguration config { get; private set; }
```

Comment:
Its public so Capital and maybe a name change to ConfigReader or ConfigStore

Reply:
Done